app/aws/aws_mocks.py

```diff
@@ -0,0 +1,248 @@
+import json
```

Comment:
Can you remove `aws` from the filename? I feel like it may not be needed since we are in the `aws` folder